test/RecGroups.hs

```diff
@@ -33,6 +33,8 @@ pipelineOpen :: T.TestTree
 pipelineOpen =
   let correctOrder =
         [ "Foo-Helper" :| ["bar"],
+          "Foo-Helper" :| ["Bah", "fi"],
+          -- DUPLICATE!?
           "Foo-Helper" :| ["Bah", "fi"],
```

Comment:
Odd for the dependency list to have repeat elements, is this intended or does this mean there is a bug somewhere else?

Reply:
i don't think this should happen, no